usr/sbin/truenas-install

```diff
@@ -283,8 +300,10 @@ install_loader()
 	    chroot ${_mnt} grub-install --target=i386-pc /dev/${_disk}
 
 	    echo "Stamping EFI loader on: ${_disk}"
-	    chroot ${_mnt} mkdosfs -F 32 -s 1 -n EFI /dev/${_disk}2
-	    chroot ${_mnt} mount -t vfat /dev/${_disk}2 /boot/efi
+        # Passing the disk and partition to partitionLabeler for proper name format.
+        partitionLabeler _disk 2
```

Comment:
See 302, then 303 and then 304 and 305. Can you please fix the spaces / tabs here as well ?

Reply:
Sorry for being sloppy, I tried not to be! Ok I made some edits in the github editor because I have to get to work, is it looking better?